includes/admin/class-wcmypa-admin.php

```diff
@@ -1253,15 +1270,16 @@ public static function getExtraOptionsFromOrder(WC_Order $order): array
      */
     private function printDeliveryDate(DeliveryOptions $deliveryOptions): void
     {
-        if (
-            $deliveryOptions->getDate() ||
-            AbstractConsignment::PACKAGE_TYPE_PACKAGE_NAME === $deliveryOptions->getPackageType()
-        ) {
+        $deliveryDate = $deliveryOptions->getDate();
+        $deliveryType = $deliveryOptions->getDeliveryType();
```

Comment:
deze wordt niet gebruikt buiten de if, dus die mag erbinnen

Reply:
maar het maakt wel heeeeeel weinig verschil in dit geval, dus ik keur de pr goed :)